SCOPE.md

```diff
@@ -0,0 +1,13 @@
+# Scope
+
+Specifies procedures for constructing and conveying artifact trees for artifacts.  Including but not limited to: 
```

Comment:
We should use "graph" or "DAG" here, since "tree" is too specific.

Reply:
@jsgf Per the discussion yesterday I've moved the SCOPE.md to https://github.com/git-bom/spec/pull/6 .  I've also pushed an update to the PR shifting to DAGs.  Let me know what you think :)